src/main/java/dao/UserDao.java

```diff
@@ -1,4 +1,35 @@
 package dao;
 
+import model.User;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
 public class UserDao {
+
+    private List<User> users = new ArrayList<>(List.of(
+            new User(1, "Ben", true),
+            new User(2, "Alice", true),
+            new User(3, "Bill", false),
+            new User(4, "John", false)
+    ));
+
+    public void changeUserBlockStatus(int userId, boolean block) {
+        List<User> users = getUsers();
+        users.stream()
+                .filter(v -> v.getId() == userId)
+                .forEach(v -> v.setUserStatus(block));
+    }
+
+    public List<User> getUsers() {
+        return users;
+    }
+
+    public List<User> getUsersListByBlockedStatus(boolean block) {
```

Comment:
What does it mean?

Reply:
This method returns a list of users depending on the parameter passed.
If passed 'true', the method returns blocked users, if 'false', then not blocked.